mempool/clist_mempool.go

```diff
@@ -80,6 +79,14 @@ var _ Mempool = &CListMempool{}
 // CListMempoolOption sets an optional parameter on the mempool.
 type CListMempoolOption func(*CListMempool)
 
+// Lane corresponds to a transaction class as defined by the application.
+// A lane is identified by a string name and has priority level.
+// Different lanes can have the same priority.
+type Lane struct {
+	id       types.LaneID
```

Comment:
Why is `Lane` defined in the `mempool` package and the following two types in the `types` package?

Reply:
Agree to move LaneID and LanePriority to the mempool package, probably to clist_mempool.go.